src/io/aleph/dirigiste/Pool.java

```diff
@@ -469,7 +469,9 @@ public void acquire(final K key, final AcquireCallback<V> callback) {
                 }
             }
         } finally {
-            _lockedQueues.remove(key);
+            // This call is safe, there is no reason the key won't be present on the locked queues
+            // at this point.
+            _lockedQueues.compute(key, (__, v) -> v - 1);
```

Comment:
I get that this is a one-liner, but can we call `v` something a little more descriptive, like `useCount`? Or maybe `cnt` if you want to keep it short?